connectors/airwatch/src/test/java/com/vmware/connectors/airwatch/AirWatchControllerTest.java

```diff
@@ -91,7 +92,7 @@ void testDiscovery() throws Exception {
     @DisplayName("Card request success cases")
     @ParameterizedTest(name = "{index} ==> Language=''{0}''")
     @CsvSource({
-            " , success.json",
+            StringUtils.EMPTY + ", success.json",
```

Comment:
If the test case is expecting a space, then you can use ```StringUtils.SPACE``` instead of ```StringUtils.EMPTY + "```.

Reply:
Both will work same. I think `StringUtils.EMPTY` is good since it makes clear that an entirely empty value is passed to interpret as `null` reference.
 
Ref : https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-CsvSource